frontend_lib/src/component/Timeline/Timeline.jsx

```diff
@@ -173,14 +189,30 @@ export class Timeline extends React.Component {
                 >
                   {props.wysiwyg ? props.t('Simple edition') : props.t('Advanced edition')}
                 </button>
+
+                <div>
+                  <DisplayAddedFileToComment
+                    fileList={props.newCommentAsFileList}
+                    onRemoveCommentAsFile={props.onRemoveCommentAsFile}
+                  />
+                </div>
               </div>
 
               <div className={classnames(`${props.customClass}__texteditor__submit`, 'timeline__texteditor__submit')}>
+                <div>
+                  <AddFileToCommentButton
```

Comment:
`AddFileToTimeLine` perhaps? As we do not really add a file to a comment.

Reply:
fixed

Reply:
renamed to AddFileToUploadButton